Yoast/wordpress-seo
# Make open-graph naming consistent

## Summary


This PR can be summarized in the following changelog entry:

* N/A - Making naming of open graph consistent

## Relevant technical choices:

* I've renamed OpenGraph, opengraph and og in files names to `open-graph`
* For classnames I've renamed the occurences to `Open_Graph`
* For variables and class properties it has been renamed to open_graph.
* Test groups are renamed from `opengraph` to `open-graph`

I also renamed `og` to `open-graph` in the indexables tabled. For textual references in comments I have used Open Graph, this is because it's named that way
 
## Test instructions

This PR can be tested by following these steps:
* Follow the steps in https://github.com/Yoast/wordpress-seo/pull/14350 to make sure the table is build correct.
* Create a post with meta data for open graph and make sure it is present in the database
* Do the same for a term.
* Verify the right meta data is outputted on the frontend.
* Verify that Travis is happy.

## UI changes

## Documentation

## Quality assurance


Fixes #13820
